### ISSUE TITLE
[ENHANCEMENT] Delete active RDR rules when container destroyed

### ISSUE THREAD

**Issue by paulc** [BastilleBSD/bastille#178]
**Is your feature request related to a problem? Please describe.**

`bastille destroy` currently doesn't  delete any active RDR rules so these will end up still active in PF

**Describe the solution you'd like**

Clean up RDR rules on destroy

**Describe alternatives you've considered**

N/A

**Additional context**

Patch attached:

```diff
diff --git a/usr/local/share/bastille/destroy.sh b/usr/local/share/bastille/destroy.sh
index c0c1fdd..dc35fed 100644
--- a/usr/local/share/bastille/destroy.sh
+++ b/usr/local/share/bastille/destroy.sh
@@ -80,6 +80,12 @@ destroy_jail() {
             echo -e "${COLOR_GREEN}Note: jail console logs archived.${COLOR_RESET}"
             echo -e "${COLOR_GREEN}${bastille_jail_log}-$(date +%F)${COLOR_RESET}"
         fi
+
+        ## clear any active rdr rules
+        if [ ! -z "$(pfctl -a "rdr/${TARGET}" -Psn 2>/dev/null)" ]; then
+            echo -e "${COLOR_GREEN}Clearing RDR rules:${COLOR_RESET}"
+            pfctl -a "rdr/${TARGET}" -Fn
+        fi
         echo
     fi
 }
```

(I wrote the original RDR functionality but unfortunately didn't think about this at the time)

Paul

**Comment by tobiastom** [comment]
Looks like this can be closed as it has been implemented in #182.

**Comment by cedwards** [comment]
Thank you
